chapters/core/setup/index.rst

```diff
@@ -12,7 +12,10 @@ Below you will find the setup guides for various, selected environments.
    :glob:
 
    ../../install/index
```

Comment:
We could probably remove this since it's information is contained in the other links?

Reply:
You mean remove the file? I'm hesitant to do that in case it's linked from elsewhere

Reply:
Not delete, just remove from this list. But I see your point, it's up to you.

Reply:
Eh, I'll keep it for now